lib/event_tracker/jobs/development_job.rb

```diff
@@ -0,0 +1,15 @@
+require 'awesome_print'
```

Comment:
I do like `awesome_print` as much as the next person, but does this usage justify an extra dependency, having http://ruby-doc.org/stdlib-2.3.0/libdoc/pp/rdoc/PP.html already available?

Reply:
+1